### DISCUSSION TITLE
What is the accepted way to share db schema in industry?

### DISCUSSION THREAD

**Discussion by alekstsoy29** [dbeaver/dbeaver#11169] [Q&A]
I have a pretty simple schema, bunch of tables and views. Some columns have constraints like value must be between X and Y, etc.

I need to share this with others, but I can't share raw SQL script file - it needs to be in a document format (thesis). What software should I use to show the relationships and constraints?

**Comment by p91paul** [comment]
Dbeaver has an ER diagram view, you should be able to export a png from there!
